models/sudoswap/ethereum/sudoswap_ethereum_events.sql

```diff
@@ -247,25 +293,26 @@ WITH
             AND pu.minute >= '2022-4-1'
             {% endif %}
             --add in `pu.contract_address = sc.currency_address` in the future when ERC20 pairs are added in.
-        LEFT JOIN {{ source('ethereum', 'transactions') }} tx
-            ON tx.hash=sc.tx_hash
+        LEFT JOIN {{ source('ethereum', 'transactions') }} tx 
```

Comment:
similar idea here with inner join. you could either wait for other PR to merge and then merge main into your branch, or match changes

Reply:
merged in main :heavy_check_mark: